ui/src/pages/AllStories/index.tsx

```diff
@@ -229,9 +248,60 @@ export const AllStories: React.FC = () => {
     event: React.ChangeEvent<Record<string, unknown>>,
     newValue: number
   ) => {
+    cancelSearch();
     dispatch({ type: "SET_TAB_VALUE", newValue });
   };
 
+  const requestSearchHelper = (row, searchedVal: string) => {
+    let doesExist = false;
+    Object.keys(row).forEach((prop) => {
+      //Exclude search for StoryView members not displayed on table cells
+      const excludedParameters =
+        prop !== "image_url" &&
+        prop !== "video_url" &&
```

Comment:
Lets also hide the id column (i.e when i search `13` story with id 13 will also show up). Maybe we can remove the id column from the table itself.